modules/porous_flow/doc/content/modules/porous_flow/tutorial_13.md

```diff
@@ -8,6 +8,8 @@ A very simple chemical system was built in [Page 07](porous_flow/tutorial_07.md)
 - does not use an `Action` to describe the Kernels, Materials, etc.  This makes the input file quite long, but perhaps more easy to extend to multi-phase situations, different boundary conditions, etc.  An `Action` could easily be used instead, by copying the chemistry from this tutorial to that on [Page 07](porous_flow/tutorial_07.md).
 - builds a fully-saturated aqueous chemical system that could be used to describe dolomite precipitation and dissolution.
 
+This page illustrates that it is possible to build quite complicated chemical systems within PorousFlow.  However, geochemists will recognize these are still unrealistically simple, since they contain only a few species, the activity coefficients are all unity, the equilibrium constants are not easily temperature-dependent, etc.  If you require state-of-the-art geochemical modelling capability, please use MOOSE's Geochemistry module.
+
```

Comment:
Maybe add a link to the geochem docs here, if that is possible? I can never remember if we can link between modules?

Reply:
When i try this on my home computer and build just the PorousFlow doco, the link doesn't work.  When i build the entire MOOSE doco it's fine.  Probably i'm doing something wrong, but would prefer not to add the link at the moment, otherwise i keep getting errors.

Reply:
No worries